packages/wallet-backend/src/auth/auth.service.ts

```diff
@@ -39,7 +39,10 @@ export const appendAccessTokenToCookie = (
   expiresIn: number
 ) => {
   res.cookie('AccessToken', token, {
-    httpOnly: true,
+    httpOnly: env.NODE_ENV === 'production',
```

Comment:
extract cookies options in one common object/method

Reply:
well, this is for wallet-backend only. In wallet-backend-update, there's only one option for cookies already.